### PR TITLE
Create mobile skeleton: Added contact-section

### PR DESCRIPTION
**Pull request by mateo951**
- Added setup HTML skeleton for contact-section
- Modified CSS to style form-section
- Added validations to form inputs with the required attribute
- Added Formspree configuration to receive form info
- Added social media links and email to the end of contact-section
- Added linters checks
- Added hover and styling to inputs and submit bttn form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,5 +275,47 @@ <h2 class="aboutme-font font-family">Skills</h2>
...
   </section>
+  <section class="flex-container flex-title-content">
+    <h2 class="section-title font-family">I'm always interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.</h2>
+    <form class="flex-container flex-title-content flex-form" action="https://formspree.io/f/xnqlbpyj" method="post">
+      <input class="input-layout" type="text" id="name" placeholder="Full Name" name="client_name" maxlength="30" required>
+      <input class="input-layout" type="email" id="mail" placeholder="Email Address" name="client_email" required>
+      <textarea class="input-layout" id="msg" placeholder="Enter text here" name="user_message" maxlength="500" required></textarea>
+      <input class="bttn input-submit-bttn" type="submit" value="Get in touch">
+    </form>
```
Comment (Shaher-11) [inline index.html:286]: Increase the height of the message input in the form as shown below
### Original design
right

#### Your Project
wrong

File: index.html [hunk 2]
```diff
@@ -275,5 +275,47 @@ <h2 class="aboutme-font font-family">Skills</h2>
...
+            <img src="images/socialLinks/github.svg" alt="Github Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/angel.svg" alt="Angel Link">
+          </a>
+        </li>
+      </ul>
+    </nav>
```
Comment (Shaher-11) [inline index.html]: Use the footer semantic tag

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @mateo951,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:286]: Done :heavy_check_mark:

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html]: Done :heavy_check_mark:

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @mateo951 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.

May I highlight the following:
- The repo looks professional:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have a descriptive PR:heavy_check_mark:
- Your live demo link is working properly:heavy_check_mark:
- Your contact form is working properly:heavy_check_mark:

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**